src/components/NavbarInput/NavbarInput.js

```diff
@@ -1,53 +1,37 @@
 import React, { useState, useRef, useEffect } from "react";
-import axios from "axios";
-import { debounce } from "lodash";
+import { connect } from "react-redux";
+import { throttle } from "lodash";
 import { Link } from "react-router-dom";
 import {
   StyledInput,
   StyledForm,
   StyledDiv,
   StyledH5,
```

Comment:
would prefer a more meaningful name, but not a big deal

Reply:
agree. I will make sure to fix these. thank you